pkg/trace/obfuscate/testdata/json_tests.xml

```diff
@@ -617,127 +615,383 @@
+				<key>Rows Removed by Conflict Filter</key>
+				<key>Rows Removed by Filter</key>
+				<key>Rows Removed by Index Recheck</key>
+				<key>Rows Removed by Join Filter</key>
+				<key>Sampling Method</key>
+				<key>Scan Direction</key>
+				<key>Schema</key>
+				<key>Settings</key>
+				<key>Shared Dirtied Blocks</key>
+				<key>Shared Hit Blocks</key>
+				<key>Shared Read Blocks</key>
+				<key>Shared Written Blocks</key>
+				<key>Single Copy</key>
 				<key>Sort Key</key>
+				<key>Sort Method</key>
+				<key>Sort Methods Used</key>
+				<key>Sort Space</key>
+				<key>Sort Space Type</key>
+				<key>Sort Space Used</key>
+				<key>Startup Cost</key>
+				<key>Strategy</key>
+				<key>Subplan Name</key>
+				<key>Subplans Removed</key>
+				<key>Target Tables</key>
+				<key>Temp Read Blocks</key>
+				<key>Temp Written Blocks</key>
+				<key>Time</key>
+				<key>Timing</key>
+				<key>Total</key>
+				<key>Trigger</key>
+				<key>Trigger Name</key>
+				<key>Triggers</key>
+				<key>Tuples Inserted</key>
+				<key>Tuplestore Name</key>
+				<key>Total Cost</key>
+				<key>WAL Bytes</key>
+				<key>WAL FPI</key>
+				<key>WAL Records</key>
+				<key>Worker</key>
+				<key>Worker Number</key>
+				<key>Workers</key>
+				<key>Workers Launched</key>
+				<key>Workers Planned</key>
 			</KeepValues>
 			<In><![CDATA[
 {
   "Plan": {
-	"Node Type": "Unique",
-	"Parallel Aware": false,
-	"Startup Cost": 223.09,
+	"Node Type": "Aggregate",
+    "Partial Mode": "Partial",
+    "Startup Cost": 74286.07,
 	"Total Cost": 223.59,
 	"Plan Rows": 100,
 	"Plan Width": 121,
-	"Plans": [
-	  {
-		"Node Type": "Sort",
-		"Parent Relationship": "Outer",
+    "Plans": [
+      {
+		"Cache Key": "datadog 'dummy'",
+		"Conflict Arbiter Indexes": "datadog 'dummy'",
+		"Conflict Resolution": "datadog 'dummy'",
+		"Filter": "(query <> 'dogfood'::text)",
+		"Hash Cond": "(pg_stat_statements.dbid = pg_database.)",
+		"Index Cond": "(pg_stat_statements.dbid = pg_database.)",
+		"Join Filter": "datadog 'dummy'",
+		"Merge Cond": "datadog 'dummy'",
+		"Recheck Cond": "datadog 'dummy'",
+		"Sampling Parameters": ["dog", "food"],
+		"TID Cond": "datadog 'dummy'",
+		"Actual Loops": 1,
+		"Actual Rows": 2,
+		"Actual Startup Time": 3,
+		"Actual Total Time": 4,
+		"Alias": "dog",
+		"Async Capable": true,
+		"Average Sort Space Used": 1,
+		"Cache Evictions": 1,
+		"Cache Hits": 2,
+		"Cache Misses": 3,
+		"Cache Overflows": 4,
+		"Command": "sort_by_dogfood",
+		"Conflicting Tuples": 1,
+		"Constraint Name": 1,
+		"CTE Name": "Dog Food Users",
+		"Custom Plan Provider": "custom_sort_by_dogfood",
+		"Deforming": false,
+		"Emission": 1.1,
+		"Exact Heap Blocks": 1,
+		"Execution Time": 1,
+		"Expressions": false,
+		"Foreign Delete": 1,
+		"Foreign Insert": 2,
+		"Foreign Update": 3,
+		"Function Call": 1,
+		"Function Name": 2,
+		"Generation": 1.22,
+		"Group Count": 1,
+		"Group Key": 2,
+		"Grouping Sets": ["a", "b"],
+		"Hash Buckets": 1,
+		"Hash Keys": 2,
```

Comment:
I haven't looked through all of these, but can you try to find some realistic examples for the tests? The type on this one is actually incorrect: https://github.com/postgres/postgres/blob/master/src/backend/commands/explain.c#L2444-L2497

Reply:
Improved: https://github.com/DataDog/datadog-agent/pull/8411/commits/7c77d58e9dff3c11dfb740bd7e8e8fc1e709115d